### PR TITLE
fix: button id

### PR DESCRIPTION
**Pull request by maxd-nordic**

### CODE REVIEW

File: .github/workflows/build-and-release.yaml [hunk 1]
```diff
@@ -61,6 +61,7 @@ jobs:
       matrix:
         configuration:
           - thingy91.low-power.memfault
+          - thingy91.low-power.memfault.dt2c
```
Comment (coderbyheart) [inline .github/workflows/build-and-release.yaml:66]:
```suggestion
          # Double-tap to crash (dt2c) sends Button ID 42 and crashes on double tap of button
          # to demonstrate the Memfault crash reporting feature.
          - thingy91.low-power.memfault.dt2c
```

File: package.json [hunk 2]
```diff
@@ -52,242 +52,282 @@
             {
               "path": "/home/runner/work/thingy-world-firmware-aws/thingy-world-firmware-aws/thingy91.debug-*/merged.hex",
               "name": "thingy_world-Thingy91-debug-${nextRelease.gitTag}.hex",
-              "label": "Pre-build HEX file for Thingy:91 (debug)"
+              "label": "App with Bootloader HEX file for Thingy:91 (debug)"
```
Comment (coderbyheart) [inline package.json]: Please do not mix different changes in one PR.

**Review by coderbyheart — CHANGES_REQUESTED** [review]

**Review by coderbyheart — CHANGES_REQUESTED** [review]
Add section to readme about button behaviour of `dt2c` builds.

File: package.json (hunk 2 above)
Comment (maxd-nordic) [reply package.json]: okay

**Comment by maxd-nordic** [comment]
@coderbyheart ready to merge?

**Comment by maxd-nordic** [comment]
rebased

**Comment by coderbyheart** [comment]
> Add section to readme about button behaviour of `dt2c` builds.

**Comment by coderbyheart** [comment]
I've added documentation to the README.

**Pull request merged**